### PR TITLE
Format input parameters for MySQL and Oracle visitor functions

### LINKED ISSUE #8143: Format input parameters for visitor functions
### Background

Considering the check-style for functions, 
` public ASTNode visitQuerySpecification(final QuerySpecificationContext ctx) ` is expected rather than
 ` public ASTNode visitQuerySpecification(final MySQLStatementParser.QuerySpecificationContext ctx) `.

Hence, this issue aims to correct the nonstandard  `visit` function check-style.

### How?

1.  Enter into directory `~/shardingsphere/shardingsphere-sql-parser/shardingsphere-sql-parser-dialect`.
2. All the target classes in `src/main/java/org/apache/shardingsphere/sql/parser/${database Name}/visitor/`.
 - As you can see, basically, most of the function names are in the form of `visitXXX()`.
3. If you find that the input parameter is in the style of `final MySQLStatementParser.QuerySpecificationContext ctx`, please modify it to `final QuerySpecificationContext ctx`.

This issue is simply understanding, though, it will take you some of the time. :) So All the changes in one PR or a few PRs with a part of changes are both ok. 

Waiting for your joining! 😃

### PR DESCRIPTION
**Pull request by wenweibin**
Fixes #8143 .

Changes proposed in this pull request:
- Format input parameters in `MySQLFormatSQLVisitor.java`
- Format input parameters in `MySQLDDLStatementSQLVisitor.java`
- Format input parameters in `MySQLStatementSQLVisitor.java`
- Format input parameters in `OracleStatementSQLVisitor.java`

### CODE REVIEW

File: shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/java/org/apache/shardingsphere/sql/parser/oracle/visitor/statement/impl/OracleStatementSQLVisitor.java [hunk 1]
```diff
@@ -106,14 +107,14 @@
  */
 @Getter(AccessLevel.PROTECTED)
 public abstract class OracleStatementSQLVisitor extends OracleStatementBaseVisitor<ASTNode> {
-
```
Comment (kimmking) [inline shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/java/org/apache/shardingsphere/sql/parser/oracle/visitor/statement/impl/OracleStatementSQLVisitor.java:109]: keep four blank

**Review by kimmking — CHANGES_REQUESTED** [review]

File: shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/java/org/apache/shardingsphere/sql/parser/oracle/visitor/statement/impl/OracleStatementSQLVisitor.java (hunk 1 above)
Comment (wenweibin) [reply shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/java/org/apache/shardingsphere/sql/parser/oracle/visitor/statement/impl/OracleStatementSQLVisitor.java]: ok，I will add them.

**Comment by tristaZero** [comment]
Oh, thanks for your TWO successive PRs. 👍

**Review by tristaZero — APPROVED** [review]
Fine with me. :)

**Review by kimmking — APPROVED** [review]

**Pull request merged**